src/kvilib/ext/KviCryptEngine.cpp

```diff
@@ -45,62 +45,62 @@
 		[big]Introduction[/big][br]
 		Well, there is a concrete background noise about security around the net.
 		And I've thought that adding a little cryptography support to KVIrc wasn't
-		a bad idea. So I've first comed out with the "crypt engines", that
-		allowed to have secure conversations on channels, queries and dcc chats;
+		a bad idea. So I first came out with the "crypt engines", that
```

Comment:
`So I first came out with the "crypt engines" that allowed secure conversations in channels, queries and DCC chats;`